src/test/java/org/zalando/nakadi/controller/SchemaControllerTest.java

```diff
@@ -37,8 +44,52 @@ public void testFailure() {
         Mockito.when(schemaService.getSchemas("et_test_name", 0, 1))
                 .thenReturn(Result.problem(Problem.valueOf(Response.Status.SERVICE_UNAVAILABLE)));
         final ResponseEntity<?> result =
-                new SchemaController(schemaService).getSchemas("et_test_name", 0, 1, nativeWebRequest);
+                new SchemaController(schemaService, eventTypeService)
+                        .getSchemas("et_test_name", 0, 1, nativeWebRequest);
         Assert.assertEquals(HttpStatus.SERVICE_UNAVAILABLE,  result.getStatusCode());
     }
 
+    @Test
+    public void testGetLatestSchemaVersionThen200() {
+        final EventType eventType = buildDefaultEventType();
+        Mockito.when(eventTypeService.get(eventType.getName())).thenReturn(Result.ok(eventType));
+        final ResponseEntity<?> result =
+                new SchemaController(schemaService, eventTypeService)
+                        .getSchemaVersion(eventType.getName(), "latest", nativeWebRequest);
+        Assert.assertEquals(HttpStatus.OK, result.getStatusCode());
```

Comment:
I think it's fine to use assertEquals, but we use extensively hamcrest matchers with `assertThat` in other test classes.